packages/i18n/src/utils/key.ts

```diff
@@ -13,6 +13,17 @@ export function extractI18nKey(input: string, key = 'i18n'): string {
     return input.replace(regPattern, '').replace(/\}$/gm, '').trim();
 }
 
+/**
+ * Extracts double curly brackets key from the given input.
+ *
+ * @param input string to extract the double curly brackets key from
+ * @returns extracted key
+ */
+export function extractDoubleCurlyBracketsKey(input: string): string {
+    const data = input.trim();
```

Comment:
Hi @marufrasully, could we combine this with a check whether the key starts with `{{` and ends with `}}`. Otherwise all consumers need to check if it is a double curly bracket key.